cypress/e2e/expenses/Delete.cy.js

```diff
@@ -0,0 +1,90 @@
+import '@testing-library/cypress/add-commands';
+
+import {
+  getUKFormattedEuros,
+  getUKFormattedDate,
+} from '../../../src/utils/helpers';
+
+describe('DELETE EXPENSE - NOT CURRENT MONTH', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', { fixture: 'expense-array' }).as(
+      'getExpenseArray'
+    );
+    cy.visit('/');
+    cy.wait('@getExpenseArray');
+  });
+
+  beforeEach(() => {
+    cy.intercept('DELETE', '**/items/**', {}).as('deleteExpense');
```

Comment:
Same as above, it could be moved inside one `it()` block but we would loose the overview at the top of the page on which intercepts we are running (which I kinda really like)